jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Server.java

```diff
@@ -86,7 +86,7 @@ public Server(
 			@JsonProperty("queries") List<Query> queries) {
 		this.alias = resolveProps(alias);
 		this.host = resolveProps(host);
-		this.port = port;
+		this.port = resolveProps(port);
```

Comment:
sure.. I added a unit test to the ServerTests which compares a server created with system variables and one created with plain variables. It failes if they don't match.